rotkehlchen/db/dbhandler.py

```diff
@@ -3422,13 +3424,16 @@ def delete_web3_node(self, identifier: int) -> None:
                 exclude_identifier=None,
             )
 
-    def get_user_notes(self, filter_query: UserNotesFilterQuery) -> List[UserNote]:
+    def get_user_notes(
+        self,
+        filter_query: UserNotesFilterQuery,
+        cursor: 'DBCursor',
```

Comment:
Please for the love of god configure your editor to properly indent those. At some point we will get a styling linter but until we do just do it. I have asked it repeatedly.